tests/integration/attendanceSheets.test.js

```diff
@@ -230,11 +230,69 @@ describe('ATTENDANCE SHEETS ROUTES - POST /attendancesheets', () => {
       sinon.assert.calledTwice(sendNotificationToUser);
     });
 
-    it('should  return 400 if single course but slot is missing', async () => {
+    it('should upload trainer signature and create attendance sheet for inter course (mobile)', async () => {
+      const slots = [slotsList[12]._id.toHexString(), slotsList[13]._id.toHexString()];
+      const attendanceSheetsLengthBefore = await AttendanceSheet.countDocuments({ course: coursesList[1]._id });
+      const formData = {
+        course: coursesList[1]._id.toHexString(),
+        signature: 'test',
+        trainees: coursesList[1].trainees[0].toHexString(),
+        origin: MOBILE,
+        trainer: trainer._id.toHexString(),
+      };
+
+      const form = generateFormData(formData);
+      slots.forEach(slot => form.append('slots', slot));
+      uploadCourseFile.returns({ publicId: '1234567890', link: 'https://test.com/signature.pdf' });
+
+      const response = await app.inject({
+        method: 'POST',
+        url: '/attendancesheets',
+        payload: getStream(form),
+        headers: { ...form.getHeaders(), Cookie: `alenvi_token=${authToken}` },
+      });
+
+      expect(response.statusCode).toBe(200);
+      const attendanceSheetsLengthAfter = await AttendanceSheet.countDocuments({ course: coursesList[1]._id });
+      expect(attendanceSheetsLengthAfter).toBe(attendanceSheetsLengthBefore + 1);
+      sinon.assert.calledOnce(uploadCourseFile);
+      sinon.assert.calledTwice(sendNotificationToUser);
+    });
+
+    it('should upload trainer signature and update attendance sheet for inter course (mobile)', async () => {
```

Comment:
Le titre de ce test me semble incohérent, on upload pas la signature. Est-ce que ce serait pas plutot : "should get existing trainer signature and update attendance sheet for inter course (mobile)" ?